docs/examples.md

```diff
@@ -441,3 +441,73 @@ new BaseJobBuilder(
 }
 ```
 
+
+### Using GHE PR Watcher
+
+Create a job that builds pull requests for a specific repo in a
+GitHub Enterprise instance. The `<unique github credential UUID>` should be
+replaced with a credential ID created by the GitHub Pull Request Builder
+itself. This can be found in the `config.xml` of a job already configured
+to use the builder or from the `org.jenkinsci.plugins.ghprb.GhprbTrigger.xml`
+in a Jenkins installation.
+
+
+#### For GitHub Enterprise
+
+```groovy
+import jenkins.automation.builders.BaseJobBuilder
+import jenkins.automation.utils.GheUtils
```

Comment:
@jacob is this referring to the old one?

Reply:
Good catch.